src/SDKs/Monitor/Management.Monitor/Microsoft.Azure.Management.Monitor.csproj

```diff
@@ -1,15 +1,68 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.reference.props'))" />
   <PropertyGroup>
-    <PackageId>Microsoft.Azure.Management.Monitor</PackageId>
     <Description>Microsoft Azure Monitor Library</Description>
     <VersionPrefix>0.16.0-preview</VersionPrefix>
-    <AssemblyName>Microsoft.Azure.Management.Monitor</AssemblyName>
+    <AssemblyName>Microsoft.Azure.Monitor</AssemblyName>
+
+    <PackageId>Microsoft.Azure.Management.Monitor</PackageId>
```

Comment:
@gucalder package name and assembly name should match, unless you have a reason to separate them out.